commandapi-platforms/commandapi-bukkit/commandapi-bukkit-nms/commandapi-bukkit-1.21.4/src/main/java/dev/jorel/commandapi/nms/NMS_1_21_R3.java

```diff
@@ -1011,7 +1025,17 @@ public final void reloadDataPacks() {
 			this.<MinecraftServer>getMinecraftServer().resources.close();
 			this.<MinecraftServer>getMinecraftServer().resources = serverResources;
 			this.<MinecraftServer>getMinecraftServer().server.syncCommands();
-			this.<MinecraftServer>getMinecraftServer().getPackRepository().setSelected(collection);
+			if (!setSelectedWithBoolean) {
+				this.<MinecraftServer>getMinecraftServer().getPackRepository().setSelected(collection);
+			} else {
+				try {
+					minecraftServerSetSelected.invoke(this.<MinecraftServer>getMinecraftServer().getPackRepository(), collection, true);
+				} catch (Throwable e) {
+					for (StackTraceElement stackTraceElement : e.getStackTrace()) {
+						CommandAPI.logError(stackTraceElement.toString());
+					}
```

Comment:
The `CommandAPILogger` class does have the [`#severe(String message, Throwable exception)`](https://github.com/CommandAPI/CommandAPI/blob/37f96b59d000ce661629410af24624fc1d4f737f/commandapi-core/src/main/java/dev/jorel/commandapi/CommandAPILogger.java#L56) method which could be used here. That method is used by [`CommandAPIExecutor`](https://github.com/CommandAPI/CommandAPI/blob/37f96b59d000ce661629410af24624fc1d4f737f/commandapi-core/src/main/java/dev/jorel/commandapi/CommandAPIExecutor.java#L81) twice, although it has to call `CommandAPI.getLogger()` since the `CommandAPI` class doesn't have a logging method that takes an exception. So maybe we'd also want to add a `CommandAPI#severe(String message, Throwable exception)` to use here and in `CommandAPIExecutor`?